src/screen.ts

```diff
@@ -16,6 +16,12 @@ const defaultScreen: RenderResult = {
   get container(): ReactTestInstance {
     throw new Error(SCREEN_ERROR);
   },
+  get root(): ReactTestInstance {
```

Comment:
Is this type accurate? Isn't there a narrower type for host elements?

Reply:
unfortunately no, though we might think about something like that in the future